### PR TITLE
onClick + ref was added

### PR DESCRIPTION
**Pull request by VladSmaug**
Added onclick and unworked ref

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -7,17 +7,21 @@ import Dialogs from "./components/Dialogs";
 
 import "./App.css";
 
-import { USERS_LIST, POST_DATA, USERS_DATA } from "./utils/static/constants";
-
-function App() {
+function App(props) {
```
Comment (Bohdan-Zavhorodskyi) [inline src/App.js]: Це можна перетворити в arrow function, підхід має бути один тому так, перепиши на arrow function

File: src/components/Dialogs/index.jsx [hunk 2]
```diff
@@ -29,6 +29,16 @@ const MessageItem = (props) => {
   return <div className={styles.message}>{text}</div>;
 };
 
+const areaRef = createRef();
+const onClickHandler = () => {
```
Comment (Bohdan-Zavhorodskyi) [inline src/components/Dialogs/index.jsx:34]: Add space between constants/functions

File: src/components/Dialogs/index.jsx [hunk 3]
```diff
@@ -29,6 +29,16 @@ const MessageItem = (props) => {
...
+  const areaValue = areaRef.current?.value;
+  alert(areaValue);
+};
+const onChangeHandler = (event) => {
+  console.log(event.target.value);
+};
+
+console.log(areaRef, "areaRef");
```
Comment (Bohdan-Zavhorodskyi) [inline src/components/Dialogs/index.jsx]: remove unnecessary console.log

File: src/components/Dialogs/index.jsx [hunk 4]
```diff
@@ -47,8 +57,16 @@ const Dialogs = ({ list }) => {
...
+      <button onClick={onClickHandler}>Submit</button>{" "}
+      <input type="text" onChange={onChangeHandler} />
     </div>
   );
 };
 
+/* onchange;
+onsubmit;
+onclick;
+onreset; */
```
Comment (Bohdan-Zavhorodskyi) [inline src/components/Dialogs/index.jsx]: do we need this commented code?

File: src/components/Profile/MyPost/index.jsx [hunk 5]
```diff
@@ -1,16 +1,28 @@
 import Post from "./Post";
 
 import styles from "./MyPost.module.css";
+import { createRef } from "react";
```
Comment (Bohdan-Zavhorodskyi) [inline src/components/Profile/MyPost/index.jsx]: React imports should be at the top of imports

File: src/redux/state.js [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+  {
+    avatar: (
+      <img
+        src="\images\pngtree-human-avatar-free-vector-png-image_4825373.png"
+        alt="avatar"
+      />
+    ),
+    text: "lorem",
+  },
+];
```
Comment (Bohdan-Zavhorodskyi) [inline src/redux/state.js]: Do we need these constants above after adding a state with the same data?

**Review by Bohdan-Zavhorodskyi — CHANGES_REQUESTED** [review]

File: src/components/Dialogs/index.jsx (hunk 4 above)
Comment (VladSmaug) [reply src/components/Dialogs/index.jsx]: Was deleted

**Review by Bohdan-Zavhorodskyi — APPROVED** [review]

**Pull request merged**